test/Main.hs

```diff
@@ -27,6 +28,7 @@ main = do
     selected <- getArgs
     let go current = when (null selected || current `elem` selected)
     go "AcyclicAdjacencyMap"  testAcyclicAdjacencyMap
+    go "AcyclicOrd"           testAcyclicOrd
```

Comment:
I think this should belong to the `AcyclicAdjacencyMap` testsuite to avoid having too many modules.